src/main/app/forms/models/multiRowForm.ts

```diff
@@ -0,0 +1,43 @@
+import { Serializable } from 'models/serializable'
+import { ItemInMultiRowForm } from 'forms/models/itemInMultiRowForm'
+
+export abstract class MultiRowForm<T extends ItemInMultiRowForm> implements Serializable<MultiRowForm<T>> {
```

Comment:
If this is meant to be reusable, I think it would make sense to have some unit tests for it.